presto-main/src/test/java/io/prestosql/sql/planner/iterative/TestMemo.java

```diff
@@ -238,8 +238,8 @@ public void testEvictCostOnReplace()
         Memo memo = new Memo(idAllocator, x);
         int xGroup = memo.getRootGroup();
         int yGroup = getChildGroup(memo, memo.getRootGroup());
-        PlanCostEstimate yCost = new PlanCostEstimate(42, 0, 0);
-        PlanCostEstimate xCost = new PlanCostEstimate(42, 0, 37);
+        PlanCostEstimate yCost = new PlanCostEstimate(42, 0, 0, 0);
```

Comment:
could we have some (very basic) tests cases

Reply:
That's good idea, after someone approves what i have so far.

Reply:
Yes. It looks good